client-src/elements/form-field-enums.ts

```diff
@@ -396,10 +400,10 @@ export const GATE_TYPES = {
   TESTING_PLAN: 75,
 };
 
-export const GATE_PREPARING = 0;
-export const GATE_REVIEW_REQUESTED = 2;
-export const GATE_NA_REQUESTED = 9;
-export const VOTE_OPTIONS = {
+export const GATE_PREPARING: number = 0;
```

Comment:
These explicit types, when the type is obvious, are considered redundant and unnecessary in google3, so I wonder if the lint script (whatever we are using now) complained about the lack of the type here, and if so, should we change to configuration not complain.    I'm not sure about other cases where the value of a constant is assigned immediately given a literal or constructor that is clear, e.g. Set<string> and Record<string, number>.

Reply:
it did not complain... do you want me to remove them in the current files? I'm not sure about cases like Set and Record and would not those redundant annotations in further PRS

Reply:
I think it should be fine to remove explicit types where both humans and tools can easily figure out the inferred types, and especially when they are constants.   So it still depends on how smart the tools are, which I don't know.

Reply:
should we keep it for now then?

Reply:
I think we should remove these simple redundant types, and only include types when either Typescript demands it or it helps a human reader.